### PR TITLE
Added notes about project-member ability to delete ns

### LINKED ISSUE #3704: Document that project members inherit kubernetes edit role and ability to delete entire namespace
```
 it sounds like the client was not expecting the users to be able to delete namespace in this situation
```

```
On the issue itself, at a glance it sounds like "functions as designed" and may want to improve documentation. "Project Member" role inherits from "Kubernetes edit" role and this is a built-in role that allows namespaces management, including deletion.
```

JIRA https://jira.suse.com/browse/SURE-2320?page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel&focusedCommentId=1137837#comment-1137837

### PR DESCRIPTION
**Pull request by jtravee**
Closes #3704

### CODE REVIEW

File: content/rancher/v2.5/en/monitoring-alerting/rbac/_index.md [hunk 1]
```diff
@@ -129,7 +129,11 @@ The relationship between the default roles deployed by Rancher Cluster Manager (
 | cluster-owner | cluster-admin | N/A | ClusterRoleBinding |
 | cluster-member | admin | monitoring-admin | ClusterRoleBinding |
 | project-owner | edit | monitoring-admin | RoleBinding within Project namespace |
-| project-member | view | monitoring-edit | RoleBinding within Project namespace |
+| project-member (see note below) | view | monitoring-edit | RoleBinding within Project namespace |
+
+>**Note:**
+>
+>By default, the Rancher role of `project-member` inherits from the `Kubernetes-Edit` role. As such, the `project-member` role is a built-in role that will allow namespace management, including the ability to delete namespaces. For more information about project roles, click [here]({{<baseurl>}}/rancher/v2.5/en/admin-settings/rbac/cluster-project-roles/#project-roles).
```
Comment (snasovich) [inline content/rancher/v2.5/en/monitoring-alerting/rbac/_index.md:136]: `project-owner` will have the ability to delete namespaces too.
In fact I believe this table is wrong as `Project Owner` inherits from `admin` k8s role and `Project Member` inherits from `edit` per https://github.com/rancher/rancher/blob/ec31e6456d4068899460bb83640b63c83eaa72c6/pkg/data/management/role_data.go#L234-L309 (see `setRoleTemplateNames` calls).

**Review by snasovich — CHANGES_REQUESTED** [review]
Have one comment that I believe needs to be addressed.

File: content/rancher/v2.5/en/monitoring-alerting/rbac/_index.md (hunk 1 above)
Comment (jtravee) [reply content/rancher/v2.5/en/monitoring-alerting/rbac/_index.md]: Updated table and note as per feedback.

File: content/rancher/v2.5/en/project-admin/project-members/_index.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ Following project creation, you can add users as project members so that they ca
 
     >**Notes:**
     >
-    >- Users assigned the `Owner` or `Member` role for a project automatically inherit the `namespace creation` role. However, this role is a [Kubernetes ClusterRole](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-and-clusterrole), meaning its scope extends to all projects in the cluster. Therefore, users explicitly assigned the `Owner` or `Member` role for a project can create namespaces in other projects they're assigned to, even with only the `Read Only` role assigned.
+    >- Users assigned the `Owner` or `Member` role for a project automatically inherit the `namespace creation` role. However, this role is a [Kubernetes ClusterRole](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-and-clusterrole), meaning its scope extends to all projects in the cluster. Therefore, users explicitly assigned the `Owner` or `Member` role for a project can create or delete namespaces in other projects they're assigned to, even with only the `Read Only` role assigned.
```
Comment (btat) [inline content/rancher/v2.5/en/project-admin/project-members/_index.md:44]: This same note is used on other pages too.

File: content/rancher/v2.5/en/monitoring-alerting/rbac/_index.md [hunk 3]
```diff
@@ -128,8 +128,12 @@ The relationship between the default roles deployed by Rancher Cluster Manager (
...
 | cluster-owner | cluster-admin | N/A | ClusterRoleBinding |
 | cluster-member | admin | monitoring-admin | ClusterRoleBinding |
-| project-owner | edit | monitoring-admin | RoleBinding within Project namespace |
-| project-member | view | monitoring-edit | RoleBinding within Project namespace |
+| project-owner* | admin | monitoring-admin | RoleBinding within Project namespace |
+| project-member* | edit | monitoring-edit | RoleBinding within Project namespace |
+
+>**Note:**
+>
+>By default, the Rancher role of `project-member` inherits from the `Kubernetes-edit` role, and the `project-owner` role inherits from the `Kubernetes-admin` role. As such, both `project-member` and `project-owner` roles will allow for namespace management, including the ability to create and delete namespaces. For more information about project roles, click [here]({{<baseurl>}}/rancher/v2.5/en/admin-settings/rbac/cluster-project-roles/#project-roles).
```
Comment (btat) [inline content/rancher/v2.5/en/monitoring-alerting/rbac/_index.md:136]: I don't think the monitoring/alerting page is the appropriate page for this. Updates should be made to https://rancher.com/docs/rancher/v2.5/en/admin-settings/rbac/cluster-project-roles/ as it has tables that define the capabilities for the built-in {cluster, project}-{owner, member} roles.

File: content/rancher/v2.6/en/monitoring-alerting/rbac/_index.md [hunk 4]
```diff
@@ -124,8 +124,12 @@ The relationship between the default roles deployed by Rancher (i.e. cluster-own
...
 | cluster-owner | cluster-admin | N/A | ClusterRoleBinding |
 | cluster-member | admin | monitoring-admin | ClusterRoleBinding |
-| project-owner | edit | monitoring-admin | RoleBinding within Project namespace |
-| project-member | view | monitoring-edit | RoleBinding within Project namespace |
+| project-owner* | admin | monitoring-admin | RoleBinding within Project namespace |
+| project-member* | edit | monitoring-edit | RoleBinding within Project namespace |
+
+>**Note:**
+>
+>By default, the Rancher role of `project-member` inherits from the `Kubernetes-edit` role, and the `project-owner` role inherits from the `Kubernetes-admin` role. As such, both `project-member` and `project-owner` roles will allow for namespace management, including the ability to create and delete namespaces. For more information about project roles, click [here]({{<baseurl>}}/rancher/v2.5/en/admin-settings/rbac/cluster-project-roles/#project-roles).
```
Comment (btat) [inline content/rancher/v2.6/en/monitoring-alerting/rbac/_index.md:132]: See above.

File: content/rancher/v2.6/en/project-admin/project-members/_index.md [hunk 5]
```diff
@@ -38,7 +38,7 @@ Following project creation, you can add users as project members so that they ca
 
     >**Notes:**
     >
-    >- Users assigned the `Owner` or `Member` role for a project automatically inherit the `namespace creation` role. However, this role is a [Kubernetes ClusterRole](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-and-clusterrole), meaning its scope extends to all projects in the cluster. Therefore, users explicitly assigned the `Owner` or `Member` role for a project can create namespaces in other projects they're assigned to, even with only the `Read Only` role assigned.
+    >- Users assigned the `Owner` or `Member` role for a project automatically inherit the `namespace creation` role. However, this role is a [Kubernetes ClusterRole](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-and-clusterrole), meaning its scope extends to all projects in the cluster. Therefore, users explicitly assigned the `Owner` or `Member` role for a project can create or delete namespaces in other projects they're assigned to, even with only the `Read Only` role assigned.
```
Comment (btat) [inline content/rancher/v2.6/en/project-admin/project-members/_index.md:41]: See above.

**Review by btat — CHANGES_REQUESTED** [review]

File: content/rancher/v2.5/en/project-admin/project-members/_index.md (hunk 2 above)
Comment (jtravee) [reply content/rancher/v2.5/en/project-admin/project-members/_index.md]: Created a new note on this page and the other 2 pages as suggested to advise of capabilities.

File: content/rancher/v2.5/en/monitoring-alerting/rbac/_index.md (hunk 3 above)
Comment (jtravee) [reply content/rancher/v2.5/en/monitoring-alerting/rbac/_index.md]: Removed from rbac page, added here (and as per note, added also to the two other pages that may reference the roles/permissions).

File: content/rancher/v2.6/en/monitoring-alerting/rbac/_index.md (hunk 4 above)
Comment (jtravee) [reply content/rancher/v2.6/en/monitoring-alerting/rbac/_index.md]: Revised per feedback.

File: content/rancher/v2.5/en/monitoring-alerting/rbac/_index.md (hunk 3 above)
Comment (jtravee) [reply content/rancher/v2.5/en/monitoring-alerting/rbac/_index.md]: Removed from RBAC page, added to the 3 pages that reference these project roles/permissions to ensure the user knows the capabilities.

File: content/rancher/v2.6/en/project-admin/project-members/_index.md (hunk 5 above)
Comment (jtravee) [reply content/rancher/v2.6/en/project-admin/project-members/_index.md:41]: Removed from RBAC page, added to the 3 pages that reference these project roles/permissions to ensure the user knows the capabilities.

File: content/rancher/v2.6/en/monitoring-alerting/rbac/_index.md (hunk 4 above)
Comment (jtravee) [reply content/rancher/v2.6/en/monitoring-alerting/rbac/_index.md]: Removed from RBAC page, added to the 3 pages that reference these project roles/permissions to ensure the user knows the capabilities.

File: content/rancher/v2.5/en/project-admin/project-members/_index.md (hunk 2 above)
Comment (jtravee) [reply content/rancher/v2.5/en/project-admin/project-members/_index.md]: Removed from RBAC page, added to the 3 pages that reference these project roles/permissions to ensure the user knows the capabilities.

**Pull request merged**